core/src/ConfirmationModal.html

```diff
@@ -62,7 +76,14 @@ <h3 class="fd-title fd-title--h5">{settings.header}</h3>
       buttonDismiss: LuigiI18N.getTranslation('luigi.button.dismiss'),
       buttonConfirm: LuigiI18N.getTranslation('luigi.button.confirm')
     };
-
+    const sanitizedHtml = EscapingHelpers.sanitizeHtml(settings.body);
+    settings = {
+      ...settings,
+      body: EscapingHelpers.restoreSanitizedTags(
+        sanitizedHtml,
+        restoreSanitizedTags
+      )
+    };
```

Comment:
Should not be better to just create a new method in EscapingHelpers.sanatizeHtmlWithSpecialChar (or a better name) where you can inject the whole logic, instead of putting it in ConfirmationModal.html? 
I guess that also the restoreSanitizedTags Array should be moved to EscapingHelpers.js (I think we could reuse it in other element, like Alert or whatever... )